modules/tfm/zephyr/CMakeLists.txt

```diff
@@ -30,6 +30,13 @@ else()
   endif()
 endif()
 
+if (CONFIG_SETTINGS_FCB OR CONFIG_SETTINGS_NVS)
```

Comment:
Shouldn't it depend on `CONFIG_NVS` rather than `CONFIG_SETTINGS_FCB `OR `CONFIG_SETTINGS_NVS`.

Reply:
It's the same condition as used in `partition_manager.cmake` for including `pm.yml.settings` which defines the settings partition. Doesn't it work in some scenario?

Reply:
Hi. Thanks for the super fast feedback. We are using CONFIG_NVS directly in our application (without settings or fcb). So in that case it does not work. Adding CONFIG_SETTINGS_NVS solves the problem but it's not so nice to add stuff that we don't need in terms of ressource usage. Do you think this could be changed?

Reply:
We can't change this condition alone. We also need to update `flash_layout.h` + `region_defs.h` to refer to the nvs_storage partition instead of settings_storage in case `CONFIG_NVS=y AND CONFIG_SETTINGS_NVS=n`. But it's a valid point we should support both cases. I'll create a task for that and get back to you when it's ready (I can't provide you with ETA though).